packages/contracts/scripts/set-scalar.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+RPC_URL=${RPC_URL:-http://localhost:8545}
+OVM_GAS_ORACLE=0x420000000000000000000000000000000000000F
+
+SCALAR=${SCALAR:-1000000}
+
+if [[ -z $PRIVATE_KEY ]]; then
+    echo "Must set PRIVATE_KEY env var"
+    exit 1
+fi
+
+cast send --rpc-url $RPC_URL \
+    --private-key $PRIVATE_KEY \
+    --legacy \
+    --gas-price 0 \
+    $OVM_GAS_ORACLE \
+    'setScalar(uint256)()' \
```

Comment:
Shouldn't this be `setScalar(uint256)`?

Reply:
Since nothing is returned, its a noop on including it

Reply:
i pushed a commit to remove it

Reply:
ah ok. I wasn't sure if cast was doing something dumb like running keccak on the arg, since that'll yield a different signature.

Reply:
the second `()` will format the return value and won't be included in calculating the function selector